library/core/class.webscraper.php

```diff
@@ -30,17 +30,17 @@ class WebScraper {
     public function __construct() {
         // Add some default sites.
         $this->registerType('getty', ['embed.gettyimages.com'], [$this, 'lookupGetty']);
-        $this->registerType('hitbox', ['hitbox.tv'], [$this, 'lookupHitbox']);
-        $this->registerType('imgur', ['i.imgur.com'], [$this, 'lookupImgur']);
+        $this->registerType('imgur', ['imgur.com'], [$this, 'lookupImgur']);
         $this->registerType('instagram', ['instagram.com', 'instagr.am'], [$this, 'lookupInstagram']);
-        $this->registerType('pinterest', ['pinterest.com'], [$this, 'lookupPinterest']);
+        $this->registerType('pinterest', ['pinterest.com', 'pinterest.ca'], [$this, 'lookupPinterest']);
+        $this->registerType('smashcast', ['hitbox.tv', 'smashcast.tv'], [$this, 'lookupSmashcast']);
         $this->registerType('soundcloud', ['soundcloud.com'], [$this, 'lookupSoundcloud']);
         $this->registerType('twitch', ['twitch.tv'], [$this, 'lookupTwitch']);
         $this->registerType('twitter', ['twitter.com'], [$this, 'lookupTwitter']);
         $this->registerType('vimeo', ['vimeo.com'], [$this, 'lookupVimeo']);
         $this->registerType('vine', ['vine.co'], [$this, 'lookupVine']);
         $this->registerType('wistia', ['wistia.com', 'wi.st'], [$this, 'lookupWistia']);
-        $this->registerType('youtube', ['youtube.com', 'youtu.be'], [$this, 'lookupYouTube']);
+        $this->registerType('youtube', ['youtube.com', 'youtube.ca', 'youtu.be'], [$this, 'lookupYouTube']);
```

Comment:
No that's not enough but a bit out of scope. I created a separate issue for it: https://github.com/vanilla/vanilla/issues/6583